frontend/app/embed.ts

```diff
@@ -203,9 +214,9 @@ function init() {
       document.body.appendChild(this.node);
       document.addEventListener('keydown', this.onKeyDown);
       setTimeout(() => {
-        this.back!.setAttribute('data-animation', '');
-        this.node!.setAttribute('data-animation', '');
-        this.iframe!.focus();
+        this.back?.setAttribute('data-animation', '');
```

Comment:
I see, so it would probably become,

```ts
      setTimeout(() => {
        this.back!.setAttribute('data-animation', '');
        this.node!.setAttribute('data-animation', '');
        this.iframe!.focus();
      }, 400);
```